packages/appium/package.json

```diff
@@ -41,9 +41,10 @@
     "types"
   ],
   "scripts": {
-    "build:docs": "node ./scripts/parse-yml-commands.js",
+    "build:docs": "node docs/scripts/build-docs.js",
```

Comment:
ok, so I should probably close #17909?

Reply:
correct. the parse-yml-commands thing is obsolete (not sure why it got put back in your earlier commit), but `build:docs` and `publish:docs` are in active use.